website/src/plugins/github-release-plugin.ts

```diff
@@ -0,0 +1,178 @@
+  return {
+    name: 'docusaurus-plugin-github-release',
+
+    async loadContent(): Promise<GlobalData> {
+      console.log('Fetching latest GitHub release for podman-desktop using Octokit...');
+
+      const octokit = new Octokit();
+
+      try {
+        const { data: releaseData } = await octokit.rest.repos.getLatestRelease({
+          owner: 'containers',
+          repo: 'podman-desktop',
+        });
+
+        const { tag_name, assets } = releaseData;
+
+        if (!tag_name) {
+          throw new Error(`No tag name for latest release`);
+        }
+        const version = tag_name.replace(/^v/, ''); // Add a fallback for rawName in case it's undefined
+
+        // Helper function to find an asset or throw an error
+        function findAssetOrThrow(predicate: (asset: (typeof assets)[0]) => boolean, assetName: string): string {
+          const asset = assets.find(predicate);
+          if (!asset) {
+            throw new Error(`Required asset not found: ${assetName}`);
+          }
+          return asset.browser_download_url;
+        }
+
+        // --- Linux ---
+        const flatpakUrl = findAssetOrThrow(a => a.name.endsWith('.flatpak'), 'Linux Flatpak');
+        const linuxArmUrl = findAssetOrThrow(a => a.name.endsWith('-arm64.tar.gz'), 'Linux ARM64 .tar.gz');
+        const linuxAmdUrl = findAssetOrThrow(
+          a => a.name.endsWith('.tar.gz') && !a.name.includes('arm64'),
+          'Linux AMD64 .tar.gz',
+        );
+
+        const linuxDownloads: LinuxDownloadData = {
+          version,
+          flatpak: flatpakUrl,
+          arm64: linuxArmUrl,
+          amd64: linuxAmdUrl,
+        };
+
+        // --- macOS ---
+        const macosArm64Url = findAssetOrThrow(
+          a => a.name.endsWith('-arm64.dmg') && !a.name.includes('airgap'),
+          'macOS ARM64 DMG',
+        );
+        const macosX64Url = findAssetOrThrow(
+          a => a.name.endsWith('-x64.dmg') && !a.name.includes('airgap'),
+          'macOS x64 DMG',
+        );
+        const macosUniversalUrl = findAssetOrThrow(
+          a =>
+            a.name.endsWith('.dmg') &&
+            !a.name.includes('airgap') &&
+            a.name !== macosArm64Url.split('/').pop() &&
+            a.name !== macosX64Url.split('/').pop(),
+          'macOS Universal DMG',
+        );
+        const macosAirgapX64Url = findAssetOrThrow(
+          a => a.name.endsWith('-x64.dmg') && a.name.includes('airgap'),
+          'macOS Airgap x64 DMG',
+        );
+        const macosAirgapArm64Url = findAssetOrThrow(
+          a => a.name.endsWith('-arm64.dmg') && a.name.includes('airgap'),
+          'macOS Airgap ARM64 DMG',
+        );
+
+        const macosDownloads: MacosDownloadData = {
+          version,
+          universal: macosUniversalUrl,
+          x64: macosX64Url,
+          arm64: macosArm64Url,
+          airgapsetupX64: macosAirgapX64Url,
+          airgapsetupArm64: macosAirgapArm64Url,
+        };
+
+        // --- Windows ---
```

Comment:
```suggestion
        // --- Windows ---
```

change scheme of comment

Reply:
https://github.com/podman-desktop/podman-desktop/commit/87bc4220e48345790ff376b31a7af30f9f2f089e